core/variant/variant_internal.h

```diff
@@ -1201,176 +1201,176 @@ struct VariantInitializer<Object *> {
 };
 
 template <class T>
-struct VariantZeroAssigner {
+struct VariantDefaultInitializer {
 };
 
 template <>
-struct VariantZeroAssigner<bool> {
+struct VariantDefaultInitializer<bool> {
 	static _FORCE_INLINE_ void zero(Variant *v) { *VariantInternal::get_bool(v) = false; }
```

Comment:
If this is approved, then `zero` should also be renamed everywhere. But let's see what @reduz thinks first.